src/common/time/src/timestamp.rs

```diff
@@ -290,32 +292,50 @@ impl Timestamp {
     /// Format timestamp to ISO8601 string. If the timestamp exceeds what chrono timestamp can
     /// represent, this function simply print the timestamp unit and value in plain string.
     pub fn to_iso8601_string(&self) -> String {
+        // Safety: the format is valid
         self.as_formatted_string("%Y-%m-%d %H:%M:%S%.f%z", None)
+            .unwrap()
     }
 
     /// Format timestamp use **system timezone**.
     pub fn to_local_string(&self) -> String {
+        // Safety: the format is valid
         self.as_formatted_string("%Y-%m-%d %H:%M:%S%.f", None)
+            .unwrap()
     }
 
     /// Format timestamp for given timezone.
     /// If `tz==None`, the server default timezone will used.
-    pub fn to_timezone_aware_string(&self, tz: Option<Timezone>) -> String {
+    pub fn to_timezone_aware_string(&self, tz: Option<&Timezone>) -> String {
```

Comment:
should we also include timezone info in those formatted string?

Reply:
I'm not sure, @v0y4g3r please take a look.

Reply:
I found it's only used in MySQL protocol.